src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -369,6 +369,38 @@ public Task SortByColumnAsync(ColumnBase<TGridItem> column, SortDirection direct
         return RefreshDataAsync();
     }
 
+    /// <summary>
+    /// Sorts the grid by the specified column <paramref name="title"/> found first. If the title is not found, nothing happens.
+    /// </summary>
+    /// <param name="title">The title of the column to sort by.</param>
+    /// <param name="direction">The direction of sorting. The default is <see cref="SortDirection.Auto"/>. If the value is <see cref="SortDirection.Auto"/>, then it will toggle the direction on each call.</param>
+    public Task SortByColumnTitleAsync(string title, SortDirection direction = SortDirection.Auto)
```

Comment:
Might it be better to have the same method name, but with these two signatures?

```csharp
public Task SortColumnAsync(string title, SortDirection direction = SortDirection.Auto)
public Task SortColumnAsync(int index, SortDirection direction = SortDirection.Auto)
```

So, later we could add another way to identify the column.

Reply:
Just a personal preference, but I went ahead and changed it to overload methods. Tests have been updated as well.